### ISSUE TITLE
Bootstrap not setup 100% yet

### ISSUE THREAD

**Issue by iarobinson** [EBWiki/EBWiki#3688]
What I'm seeing is that the bootstrap isn't setup exactly yet:

<img width="1027" alt="Screen Shot 2020-09-30 at 5 03 21 PM" src="https://user-images.githubusercontent.com/7991933/94744885-50e5ea00-033f-11eb-8ca1-a568b3cd2b7d.png">

The viewport needs to be set in the head tag. Once that happens, it will look like this:

<img width="1026" alt="Screen Shot 2020-09-30 at 5 09 14 PM" src="https://user-images.githubusercontent.com/7991933/94745220-e6817980-033f-11eb-88ec-ca2fc553b11d.png">

Here is the code that needs to be added to the head HTML tag. This comes directly from [Bootstrap documentation](https://getbootstrap.com/docs/4.0/getting-started/introduction/#starter-template):
```
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

**Comment by mihrab34** [comment]
May I work on this issue

**Comment by mihrab34** [comment]
Issue #3714 - Add Viewport meta tag to head tag

**Comment by rlgreen91** [comment]
@mihrab34 Sorry, someone else is already working on this issue.

**Comment by rlgreen91** [comment]
Fix merged in #3690
